templates/federated_reporting/distributed_cleanup.py

```diff
@@ -270,6 +270,12 @@ def main():
             hostname=feeder_hostname,
         )
         response = feeder_api.status()
+        if response["status"] == 401 and sys.stdout.isatty():
+            # iteractively add feeder
+            hub_user = api.get( "user", "fr_distributed_cleanup")
+            email = hub_user['email'] or 'fr_distributed_cleanup@{}'.format(hub['ui_name'])
+            interactive_setup_feeder(hub, email, fr_distributed_cleanup_password)
+            continue
```

Comment:
Shouldn't this continue executing the code in the loop for the newly-added feeder instead of `continue` with the next feeder?